### DISCUSSION TITLE
Citations count different from the web version

### DISCUSSION THREAD

**Discussion by mallada** [scholarly-python-package/scholarly#515] [Q&A]
Hi, 

I'm using scholarly to get all the citations of a publication, but when I compare with what I see online, it seems that I'm getting less citations than I'm supposed to.

Is this some type of bug?

Thanks!

**Comment by arunkannawadi** [comment]
Can you paste a link to the specific publication you're referring to? Google Scholar has at least two different pages from what I remember, and it takes a while sometimes for the citation counts to sync between them.

**Reply by mallada** [reply]
Here's an example: https://scholar.google.com/citations?view_op=view_citation&hl=en&user=ZvRFA04AAAAJ&citation_for_view=ZvRFA04AAAAJ:tYavs44e6CUC

As of today, I get 170 citations using the api, but online there are 179.
